src/EmployerPR/SFA.DAS.EmployerPR.Api/AppStart/AddServiceCollectionExtensions.cs

```diff
@@ -44,7 +44,8 @@ public static IServiceCollection AddServiceRegistration(this IServiceCollection
         services.AddTransient<IAccountsApiClient<AccountsConfiguration>, AccountsApiClient>();
         services.AddTransient<IEmployerProfilesApiClient<EmployerProfilesApiConfiguration>, EmployerProfilesApiClient>();
         services.AddTransient<IProviderRelationshipsApiClient<ProviderRelationshipsApiConfiguration>, ProviderRelationshipsApiClient>();
-
+        services.AddTransient<IRoatpV2TrainingProviderService, RoatpV2TrainingProviderService>();
+        services.AddTransient<IRoatpCourseManagementApiClient<RoatpV2ApiConfiguration>, RoatpCourseManagementApiClient>();
```

Comment:
These two are duplicate, why do we need two anyway?

Reply:
Both are in the shared outer api, and as we discussed, the service wraps the new call.  The service consumes the ApiClient, hence both need to be configured.